pyrogram/session/session.py

```diff
@@ -359,7 +358,7 @@ async def _send(self, data: TLObject, wait_response: bool = True, timeout: float
 
         # Call log.debug twice because calling it once by appending "data" to the previous string (i.e. f"Kind: {data}")
         # will cause "data" to be evaluated as string every time instead of only when debug is actually enabled.
-        log.debug(f"Sent:")
+        log.debug("Sent:")
```

Comment:
Seeing back these lines remember me a code change that was made in https://github.com/pyrogram/pyrogram/commit/76985faa118835e1b8ee2e5fb8ce51631d90ffd4, that changes `log.debug(f"Sent:\n{message}")` to a twice log.debug call, to avoid transforming the `Message` object into a string unnecessary. But the correct fix for that issue should be:
```python
log.debug("Sent:\n%s", message)
```

instead of calling log.debug twice.

Reply:
made the suggested change!

Reply:
https://github.com/pyrogram/pyrogram/blob/a48d27f501b73eee842384723c33db298ce1e6c3/pyrogram/session/session.py#L237-L238

Can you apply it there too? And there are some log.* using f-strings which could be refactored with logging string formatter (I've found them with `grep -rEI 'log\.[a-z_]+?\(f"'`).